CHANGELOG.md

```diff
@@ -211,7 +211,7 @@ to accept the request body as an `Object`
 
 - Added options to preserve file timestamps on [file upload](./docs/files.md#upload-a-file)
 and to rename a file or preserve modification timestamp on [new version upload](./docs/files.md#upload-a-new-version-of-a-file)
-- Added [Collaboration Whitelist](./docs/collaboration-whitelist.md) functionality to allow enterprise admins
```

Comment:
Shouldn't change this because it is a history of changes.

Reply:
We should add to the CHANGELOG.md, but probably not update older history because it would be inaccurate and the commits associated with the change will no longer match the change.

Reply:
Ok, I'm reverting these